src/bundle/Resources/public/scss/_custom.scss

```diff
@@ -296,7 +296,7 @@ $modal-header-border-color: $ibexa-color-light-500;
 $modal-title-font-size: 28px;
 $modal-content-border-width: 0;
 $modal-content-border-radius: $ibexa-border-radius;
-$modal-inner-padding: 24px 0;
+$modal-inner-padding: calculateRem(24px);
```

Comment:
This has to be single value in `rem` because it is used in such expression in bootstrap scss internals:
`padding: $modal-inner-padding - $modal-footer-margin-between * .5;`
ref. https://github.com/ibexa/admin-ui-assets/blob/792cd47c43ab5cc05609e6a45242ef87934d5afb/src/bundle/Resources/public/vendors/bootstrap/scss/_modal.scss#L131
otherwise it produces:
![Screenshot 2022-02-21 at 21 09 53](https://user-images.githubusercontent.com/10233057/155020439-21127731-ec88-4ed9-aae8-1a8813ec106d.png)